src/main/java/ru/practicum/shareit/model/Request.java

```diff
@@ -1,13 +1,25 @@
 package ru.practicum.shareit.model;
 
-import lombok.Data;
+import lombok.*;
 
+import javax.persistence.*;
 import java.time.LocalDateTime;
 
-@Data
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
```

Comment:
https://projectlombok.org/features/Data
Почему ты убрал Data? Ведь это @ToString, @EqualsAndHashCode, @Getter on all fields, @Setter on all non-final fields, and @RequiredArgsConstructor

Reply:
Idea рекомендовала заменить. Using @Data for JPA entities is not recommended. It can cause severe performance and memory consumption issues.
Вернул обратно на @Data